tools/walletextension/api/staticOG/javascript.js

```diff
@@ -0,0 +1,238 @@
+const eventClick = "click";
+const eventDomLoaded = "DOMContentLoaded";
+const idJoin = "join";
+const idAddAccount = "addAccount";
+const idAddAllAccounts = "addAllAccounts";
+const idRevokeUserID = "revokeUserID";
+const idStatus = "status";
+const idUserID = "userID";
+const pathJoin = "/join/";
+const pathAuthenticate = "/authenticate/";
+const pathQuery = "/query/";
+const pathRevoke = "/revoke/";
+const obscuroChainIDDecimal = 777;
+const methodPost = "post";
+const methodGet = "get";
+const jsonHeaders = {
+    "Accept": "application/json",
+    "Content-Type": "application/json"
+};
+const metamaskRequestAccounts = "eth_requestAccounts";
+const metamaskPersonalSign = "personal_sign";
+
+function isValidUserIDFormat(value) {
+    return typeof value === 'string' && value.length === 64;
+}
+
+async function addNetworkToMetaMask(ethereum, userID, chainIDDecimal) {
+    // add network to MetaMask
+    let chainIdHex = "0x" + chainIDDecimal.toString(16); // Convert to hexadecimal and prefix with '0x'
+
+    try {
+        await ethereum.request({
+            method: 'wallet_addEthereumChain',
+            params: [
+                {
+                    chainId: chainIdHex,
+                    chainName: 'Obscuro Testnet',
+                    nativeCurrency: {
+                        name: 'Obscuro',
+                        symbol: 'OBX',
+                        decimals: 18
+                    },
+                    rpcUrls: ['http://127.0.0.1:3000/?u='+userID],
```

Comment:
Get value of server where this app was deployed (local / dev-testnet / testnet / mainnet) and connect to the correct wallet extension.